src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/ClaimStore.java

```diff
@@ -96,29 +96,29 @@ public Claim saveCountyCourtJudgement(String externalId, CountyCourtJudgment ccj
         return getClaimByExternalId(externalId);
     }
 
-    public Claim makeOffer(long claimId, Settlement settlement) {
-        logger.info(String.format("Saving offer with claim : %d", claimId));
+    public Claim makeOffer(String externalId, Settlement settlement) {
+        logger.info(String.format("Saving offer with claim : %s", externalId));
```

Comment:
Can we change this to `debug` level? Logging every successful action on `info` is a little too verbose.